index.js

```diff
@@ -225,7 +335,7 @@ async function createCodeDeployDeployment(codedeploy, clusterName, service, task
   };
   // If it hasn't been set then we don't even want to pass it to the api call to maintain previous behaviour.
   if (codeDeployDescription) {
-    deploymentParams.description = codeDeployDescription
+    deploymentParams.description = (codeDeployDescription.length <= 512) ? codeDeployDescription : `${codeDeployDescription.substring(0,511)}…`;
```

Comment:
What does the number `512` signify here, I suggest using a constant instead of a magic number.